File: app/src/main/java/com/taomish/app/android/farmsanta/farmer/models/api/cultivar/Photo.java
package com.taomish.app.android.farmsanta.farmer.models.api.cultivar;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Photo {

    @SerializedName("caption")
    @Expose
    private String caption;
    @SerializedName("fileName")
    @Expose
    private String fileName;
    @SerializedName("photoId")
    @Expose
    private String photoId;

    public String getCaption() {
        return caption;
    }

    public void setCaption(String caption) {
        this.caption = caption;
    }

    public String getFileName() {
        return fileName;
    }

    public void setFileName(String fileName) {
        this.fileName = fileName;
    }

    public String getPhotoId() {
        return photoId;
    }

    public void setPhotoId(String photoId) {
        this.photoId = photoId;
    }

}

File: app/src/main/java/com/taomish/app/android/farmsanta/farmer/models/api/pop/Photo.kt
package com.taomish.app.android.farmsanta.farmer.models.api.pop

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class Photo (
    @SerializedName("caption") @Expose val caption: String,
    @SerializedName("fileName") @Expose val fileName: String,
    @SerializedName("photoId") @Expose val photoId: String,
    @SerializedName("sequenceId") @Expose val sequenceId: Int = 0
)